pkg/btc-wallet/src/channel.js

```diff
@@ -0,0 +1,290 @@
+export default class Channel {
```

Comment:
Why does the BTC wallet have it's own implementation of this instead of using http-api?